Adding GitHub Quick Reference (#63)
bcgov/bcdg: docs/use-github-in-bcgov/quick-reference.md

- Enable [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication/) for your GitHub account
- Ensure you have an IDIR account. More details [here](bc-government-organizations-in-github.md#single-sign-on-).
- Join the `bcgov` organization following [these instructions](bc-government-organizations-in-github.md/#directions-to-sign-up-and-link-your-account-for-bcgov)
- If you have been asked to, or if you will be working on closed-source code, join the `bcgov-c` organization following [these instructions](bc-government-organizations-in-github.md/#directions-to-sign-up-and-link-your-account-for-bcgov-c-). 
- Make sure you have [Git client](https://git-scm.com/downloads/) or a code editor that supports Git installed on your computer.

## Some Resources for Learning Git & GitHub
- [About Git and GitHub](https://docs.github.com/en/get-started/start-your-journey/about-github-and-git)
- [Learn Git in 15 minutes](https://try.github.io/levels/1/challenges/1)
- [On-demand course in GitHub basics](https://skills.github.com)

## Creating a Repository (aka "repo")

> !!! note

Reviewer: ```suggestion
!!! Note
```

> 	If you are creating a repository to host code for an app that will run on OpenShift, you may want to use our [QuickStart for OpenShift Wizard](../../wizards/quickstart-for-openshift-wizard/#devhub-wizards), which will save you some steps.

Reviewer: ```suggestion
	If you are creating a repository to host code that will run on OpenShift, you may want to use our [QuickStart for OpenShift Wizard](../../wizards/quickstart-for-openshift-wizard/#devhub-wizards), which will save you some steps.
```

Reviewer: You can host and run models on OpenShift as well, so editing to be generic.

-  If you are initiating a **new repository**, follow [GitHub instructions](https://help.github.com/articles/creating-a-new-repository/) and select 'BCGov' as the owner for all new repositories and implement the [required elements](start-working-in-bcgov-github-organization.md/#initiate-new-code-or-projects)
- Pick a repository name that follows best practices, for example following a standard naming convention (e.g., using lowercase with dashes), using short and descriptive names with a prefix/suffix indicating technology stack, use case or team
- Ensure the repo contains the [mandatory files](#mandatory-files-for-all-bcgov-repositories) (LICENSE, README.md, CONTRIBUTING.md, and CODE_OF_CONDUCT.md files)
- Consider adding an About description and Topic tags to the repository details to make it easier for users to learn about the project
- Consider adding a [Project Lifecycle Badge](https://github.com/bcgov/repomountie/blob/master/doc/lifecycle-badges.md)

## Mandatory files for all `bcgov` repositories
- Every repository in `bcgov` [must have 4 files](required-pages-for-github-repository.md): 
  - LICENSE
  - README.md
  - CONTRIBUTING.md
  - CODE_OF_CONDUCT.md

> - [Sample templates](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Gov-Org-HowTo/README.md) are available for all of these 'must have' files

Reviewer: ```suggestion
- Samples templates are available for all of these 'must have' files below
```

Reviewer: Removes link to old content.

  - The content is free of [Personal Information](http://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/privacy) (Privacy)	- The content has been labelled 'Public' using the  [Information Security Classification Framework](https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security/information-security-classification) (Security)
  - The content is fully owned by the B.C. government and/or the B.C. government holds the [Intellectual Property and Copyright](https://www2.gov.bc.ca/gov/content/governments/services-for-government/policies-procedures/intellectual-property/intellectual-property-program) for the content
  - There are no other legal, contractual or policy constraints
- Read more about assessing privacy, security and intellectual property in the [BC Developer Guide](evaluate-open-source-content.md)

## Approval Requirements

> -  [Approval for publishing to GitHub](start-working-in-bcgov-github-organization.md) are from a deputy minister. Deputy ministers may choose to delegate this authority.

Reviewer: ```suggestion
-  [Approval for publishing to GitHub](start-working-in-bcgov-github-organization.md) are from a Deputy Minister. Deputy Ministers may choose to delegate this authority.
```
